packages/amplify-cli/src/domain/amplify-telemetry/NoTelemetry.ts

```diff
@@ -0,0 +1,23 @@
+import { ITelemetry } from './ITelemetry';
+
+export class NoTelemetry implements ITelemetry {
+  emitError(error: Error): Promise<void> {
+    return Promise.resolve();
+  }
+  emitInvoke(): Promise<void> {
+    return Promise.resolve();
+  }
+  emitAbort(): Promise<void> {
+    return Promise.resolve();
+  }
+  emitSuccess(): Promise<void> {
+    return Promise.resolve();
+  }
+  init(installationUuid: String, version: String, input: any): void {}
+
+  private static instance: NoTelemetry;
```

Comment:
Same as for the other `ITelemetry` implementation.

Reply:
Same as telemetry